### ISSUE TITLE
`operator` not highlighted in conversion operator definitions when conversion type is templated.

### ISSUE THREAD

**Issue by matter123** [jeff-hykin/cpp-textmate-grammar#231]
Example Code:
```c++
operator std::basic_string_view<CharT, Traits>() const noexcept {}
```

Example Image:
<img width="625" alt="Screen Shot 2019-06-09 at 14 19 22" src="https://user-images.githubusercontent.com/714007/59164457-cdf1ae00-8ac1-11e9-9257-91ad2f459c65.png">

**Comment by matter123** [comment]
This probably needs to be resolved before contexts as the grammar thinks that `std::basic_string_view` is a function call, not a function definition. When contexts are fixed it will be unhighlighted as well.

**Comment by jeff-hykin** [comment]
Yeah this will be the first step after fixing the contexts. The operator overload pattern needs to be rewritten to 1. use the block finder 2. add support for C++ literals, and now also 3. fix this issue

**Comment by jeff-hykin** [comment]
Fixed just now `v1.12.0`
